plugins/modules/web_app_pool.ps1

```diff
@@ -0,0 +1,347 @@
+#!powershell
+
+# Copyright: (c) 2015, Henrik Wallström <henrik@wallstroms.nu>
+# Copyright: (c) 2017, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+#AnsibleRequires -CSharpUtil Ansible.Basic
+
+$ErrorActionPreference = 'Stop'
+
```

Comment:
Ansible always runs with this, we can remove this as it is redundant

```suggestion
```

Reply:
@ronger4 Please split this module into a module and an _info

Reply:
Yes, will do
Work in progress

Reply:
> @ronger4 Please split this module into a module and an _info
Yes, will do
Work in progress

Reply:
> @ronger4 Please split this module into a module and an _info


Yes, will do
Work in progress